samples/app/src/main/java/com/sourcepoint/app/v6/MainActivityKotlin.kt

```diff
@@ -141,7 +141,11 @@ class MainActivityKotlin : AppCompatActivity() {
     }
 
     private fun executeCmpLib() {
-        spConsentLib.loadMessage()
+        spConsentLib.loadMessage(authId = "andre_30_08_23_19_111111") // PREVIOUS fc4f13f9-9586-4557-b509-b49c5862d5d7
+        // andre_30_08_23_18:58 ->   e389d177-8455-4a53-ba0e-b2d0f0eadf0d_23
+        // "andre_30_08_23_18:56" -> e389d177-8455-4a53-ba0e-b2d0f0eadf0d_23
+        // andre_30_08_23_19_19 ->  769297a2-183e-4f30-ad31-8d7cc27582fc
+        //
```

Comment:
Is this expected to be in the `develop`? `MainActivityKotlin` looks pretty important and I am concerned about any commented code in it

Reply:
Ohhh, thanks for catching it @Nevazhnovu , it shouldn't be there :) It's not that important though, just testing data.